hugo.toml

```diff
@@ -14,6 +14,11 @@ name = 'About'
 pageRef = '/about'
 weight = 20
 
+[[menus.main]]
+name = 'CfP'
+pageRef = 'https://pretalx.com/pyladiescon-2024/cfp'
```

Comment:
The link doesn't work on the preview site 😕

Reply:
I think we may have to hard code it (like #contact)